### PR TITLE
[SW-29] Install and configure eslint

### PR DESCRIPTION
**Pull request by svIZsollei**

### CODE REVIEW

File: src/components/MainMenu/index.js [hunk 1]
```diff
@@ -58,21 +74,36 @@ class MainMenu extends React.Component {
...
+  }
+}
+
+MainMenu.propTypes = {
+  history: PropTypes.shape(),
+};
+
+MainMenu.defaultProps = {
+  history: {
+
```
Comment (spinenco-sv) [inline src/components/MainMenu/index.js]: Please remove this space

File: src/pages/Movie/index.js [hunk 2]
```diff
@@ -1,47 +1,73 @@
...
+    this.state = {
+      title: '',
+      release_date: '',
+      director: '',
+      opening_crawl: '',
+    };
+  }
 
   componentDidMount() {
+    // console.log('blabla', this.props);
```
Comment (spinenco-sv) [inline src/pages/Movie/index.js]: Please delete console.log

**Review by spinenco-sv — CHANGES_REQUESTED** [review]

**Review by spinenco-sv — APPROVED** [review]

File: src/components/MainMenu/index.js [hunk 3]
```diff
@@ -1,32 +1,37 @@
 import React from 'react';
 import { NavLink, withRouter } from 'react-router-dom';
-import { Container, Navbar, Nav, NavDropdown } from 'react-bootstrap';
+import {
+  Container, Navbar, Nav, NavDropdown,
```
Comment (svekozak) [inline src/components/MainMenu/index.js]: @svIZsollei You can put each component name on a line of its own

File: src/components/MainMenu/index.js [hunk 4]
```diff
@@ -1,32 +1,37 @@
...
     });
-  };
-
-  handleDropdownOpen = () => {
-    this.setState({
-      dropdownOpen: true
-    });
   }
 
   handleDropdownClose = () => {
```
Comment (svekozak) [inline src/components/MainMenu/index.js]: @svIZsollei Would you make the definition of handleDropdownClose and handleDropdownOpen consequent ?

File: src/components/MainMenu/index.js [hunk 5]
```diff
@@ -35,19 +40,31 @@ class MainMenu extends React.Component {
...
+  }
+
   render() {
     const {
       movies,
       dropdownOpen,
       expanded,
     } = this.state;
-    const linkMovieList = (<span onClick={this.goToMovieList}> Star Wars Movies</span>);
+    const linkMovieList = (<span onClick={this.goToMovieList} onKeyPress={() => {}} role="link" tabIndex="0"> Star Wars Movies</span>);
```
Comment (svekozak) [inline src/components/MainMenu/index.js]: @svIZsollei @svekukucska please, look into this onKeyPress

File: src/helpers/date.js [hunk 6]
```diff
@@ -1,12 +1,9 @@
+export const isValidDate = (d) => d instanceof Date && !isNaN(d);
 export const formatedDate = (date) => {
   if (date) {
     const dateObject = new Date(date);
-    const dateFull = isValidDate(dateObject) ?
-    `${dateObject.getDate()} ${dateObject.toLocaleString('default', { month: 'long' })} ${dateObject.getFullYear()}`
-    :'';
+    const dateFull = isValidDate(dateObject) ? `${dateObject.getDate()} ${dateObject.toLocaleString('default', { month: 'long' })} ${dateObject.getFullYear()}` : '';
```
Comment (svekozak) [inline src/helpers/date.js]: @svIZsollei you can return directly the value (the local variable is redundant). Also the line length is longer than 120 characters. Please, format code.

File: src/pages/Movie/index.js [hunk 7]
```diff
@@ -1,47 +1,70 @@
...
+        this.setState({
+          title, releaseDate, director, openingCrawl,
+        });
       });
   }
 
   render() {
-    const { title, release_date, director, opening_crawl } = this.state;
+    const {
+      title, releaseDate, director, openingCrawl,
```
Comment (svekozak) [inline src/pages/Movie/index.js]: @svIZsollei each attribute on new line, please

**Review by svekozak — CHANGES_REQUESTED** [review]

**Review by svekozak — APPROVED** [review]
👍

**Pull request merged**